<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_console"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Comandos</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="comboBo_contentType"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_url"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Url</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Method</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_method"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QPushButton" name="pushButton_send">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_runMode">
         <property name="title">
          <string>Run mode</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget_runMode">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_0">
             <attribute name="title">
              <string>Device mode</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="3" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Max val</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Clock</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="3">
               <widget class="QToolButton" name="toolButton_setLocalClock">
                <property name="toolTip">
                 <string>Set local clock</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QSpinBox" name="spinBox_intervalo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
                <property name="value">
                 <number>60</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_maxVal">
                <property name="minimum">
                 <double>-9999999.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>9999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>10.000000000000000</double>
                </property>
                <property name="value">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Intervalo</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Segs.</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QDateTimeEdit" name="dateTimeEdit_clock"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_1">
             <attribute name="title">
              <string>Fixed payload</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButton_loadData">
                <property name="text">
                 <string>Cargar datos</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPlainTextEdit" name="plainTextEdit_payload"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionLimpiar_consola"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuHerramientas"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLimpiar_consola">
   <property name="text">
    <string>Limpiar consola</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
